graphql/schema/types/task.graphql

```diff
@@ -174,6 +174,7 @@ type TaskEndDetail {
   type: String!
   traceID: String
   diskDevices: [String!]!
+  failureMetadataTags: [String!]!
```

Comment:
its lowkey fine since they were already out of order but if we put this in alphabetical order it would be Awesomesauce